### PR TITLE
Change script to take multiple audiences and other claims from command line flag.

### PR DESCRIPTION
**Pull request by diemtvu**

### CODE REVIEW

**Comment by liminw** [comment]
Looks good. Can you also update README.md to reflect the change?

File: security/tools/jwt/README.md [hunk 1]
```diff
@@ -38,18 +38,17 @@ positional arguments:
...
 ```
 
-If you want to add custom claims to the JWT, you can edit sa-jwt.py, and add any claims to JWT payload
-(look for "Add any custom claims here" comment in the script).
-
 ## Example
 
 Here is an example of using sa-jwt.py to generate a JWT token.
 ```
   python sa-jwt.py /path/to/service_account.json -iss <YOUR_SERVICE_ACCOUNT_EMAIL> -aud <YOUR_AUDIENCE>
```
Comment (liminw) [inline security/tools/jwt/README.md]: The example needs to be updated.

File: security/tools/jwt/README.md [hunk 2]
```diff
@@ -50,5 +50,9 @@ optional arguments:
 
 Here is an example of using sa-jwt.py to generate a JWT token.
 ```
-  python sa-jwt.py /path/to/service_account.json -iss <YOUR_SERVICE_ACCOUNT_EMAIL> -aud <YOUR_AUDIENCE>
+  python sa-jwt.py /path/to/service_account.json -iss frod@gserviceaccount.com -aud foo,bar
```
Comment (liminw) [inline security/tools/jwt/README.md]: do you still need "python"?

Comment (diemtvu) [reply security/tools/jwt/README.md]: Removed.

**Pull request merged**